src/components/Draggable/RealDefaultDraggable.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { useDrag } from 'react-dnd'
+import { Button, HFlow, Icon } from '..'
+import { useStyles } from '../../styles'
+import { DefaultDraggableProps } from './DefaultDraggable'
+import { draggableCreateStyles } from './RealFilterDraggable'
```

Comment:
This is creating a dependency between the components. Either split these styles or put them in another file.